docs/core/diagnostics/cli-tools/dotnet-trace.md

```diff
@@ -0,0 +1,151 @@
+---
+title: dotnet-trace - .NET Core
+description: Installing and using the dotnet-trace command-line tool.
+author: sdmaclea
+ms.author: stmaclea
+ms.date: 08/21/2019
+---
+# Trace for performance analysis utility (`dotnet-trace`)
+
+The `dotnet-trace` tool is a cross-platform CLI global tool that enables the collection of .NET Core traces of a running process without any native profiler involved. It's built around the `EventPipe` technology of the .NET Core runtime as a cross-platform alternative to ETW on Windows and LTTng on Linux, which only work on a single platform. With `EventPipe`/`dotnet-trace`, we're trying to deliver the same experience on Windows, Linux, or macOS.
```

Comment:
Please rewrite this sentence to not use we. Maybe:

The goal of `dotnet-trace` is to deliver the same experience on Windows, Linux, or macOS.